src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+import task.Deadline;
+import task.Task;
+import task.Todo;
+import task.Event;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage class to manage hard drive data.
+ *
+ */
+public class Storage {
+    private String filepath;
+
+    /**
+     * duke.Storage constructor.
+     *
+     * @param filepath path of data file.
+     */
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Initially populates the Task List with the hard drive data (Input data file).
```

Comment:
There should be a blank comment line between the description and the list of tags, according to the style guide :P